R/teal_data_module.R

```diff
@@ -42,6 +46,12 @@
 #'     })
 #'   }
 #' )
+#'
+#' @name teal_data_module
+#' @rdname teal_data_module
+#'
+#' @seealso [`teal_data-class`]
```

Comment:
This plus the other `@seealso` create a new line. Ideally the documentation would be merged. I would suggest a bullet list so they look consistent while adding in different sections

![image](https://github.com/insightsengineering/teal/assets/211358/53894e39-b469-4df3-97ba-a03f21db015f)

```diff
diff --git a/R/teal_data_module-within.R b/R/teal_data_module-within.R
index c96655017..384a8c300 100644
--- a/R/teal_data_module-within.R
+++ b/R/teal_data_module-within.R
@@ -10,7 +10,8 @@
 #' @return
 #' `within` returns a `teal_data_module` object with a delayed evaluation of `expr` when the module is run.
 #'
-#' @seealso [`base::within()`], [`teal.code::within.qenv()`]
+#' @seealso - [`base::within()`]
+#' - [`teal.code::within.qenv()`]
 #'
 #' @examples
 #' tdm <- teal_data_module(
diff --git a/R/teal_data_module.R b/R/teal_data_module.R
index dd3191a91..ced92d041 100644
--- a/R/teal_data_module.R
+++ b/R/teal_data_module.R
@@ -50,7 +50,7 @@
 #' @name teal_data_module
 #' @rdname teal_data_module
 #'
-#' @seealso [`teal_data-class`]
+#' @seealso - [`teal_data-class`]
 #'
 #' @export
 teal_data_module <- function(ui, server) {
```

![image](https://github.com/insightsengineering/teal/assets/211358/940d2ef0-e2a9-446f-8768-02da94bab0f4)

Reply:
That's because the lines come from different files. Ideally they would all be put into a single line but I don't want to link to `within` from `teal_module_data` etc.
The bullet list is consistent, which is nice :+1: but a little too big IMO.

Let me think about it for a while.